docs/using.rst

```diff
@@ -155,28 +191,37 @@ Registering the Same Schema Under the Subject "Kafka-value"
 
       curl -X POST -H "Content-Type: application/vnd.schemaregistry.v1+json" \
         --data '{"schema": "{\"type\": \"string\"}"}' \
-         http://localhost:8081/subjects/Kafka-value/versions
-      {"id":1}
+         http://localhost:8081/subjects/Kafka-value/versions?pretty
+      {
+        "id" : 1
+      }
 
 -------------------------------------------------
 Fetching the Schema Again by Globally Unique ID 1
 -------------------------------------------------
 
 .. sourcecode:: bash
 
-      curl -X GET http://localhost:8081/schemas/ids/1
-      {"schema":"\"string\""}
+      curl -X GET http://localhost:8081/schemas/ids/1?pretty
+      {
+        "schema" : "\"string\""
+      }
 
-------------------------------------------------------------
-Checking if a Schema Is Registered Under Subject "Kafka-key"
-------------------------------------------------------------
+--------------------------------------------------------------
+Checking if a Schema Is Registered Under Subject "Kafka-value"
```

Comment:
The key still had version 1 and all the earlier examples update the value, so I think this was a typo.